tests/driver/api.test.ts

```diff
@@ -807,6 +807,19 @@ describe('Mongoose Model API level tests', async () => {
             assert.deepStrictEqual($vector, [1, 101]);
         });
 
+        it('supports sort() and similarity score with $meta with find()', async function() {
+            const res = await Vector.find({}, { name: 1, $similarity : 1}).sort({ $vector: { $meta: [1, 99] } });
```

Comment:
What do I need to do to get these tests to pass? I'm running locally with latest JSON API, but this test fails with the following error for me.

```
Error: Command "find" failed with the following errors: [{"message":"Unsupported projection parameter: path cannot start with '$' (no root-level operators)","errorCode":"UNSUPPORTED_PROJECTION_PARAM","exceptionClass":"JsonApiException"}]
```

Reply:
I'm running with below versions in `api-compatibility.versions` and starting the server using `bin/start_json_api.sh` and it's supported.
```
stargate_version=v2.1.0-ALPHA-6
json_api_version=v1.0.0-ALPHA-12
```